src/app/ui-library/inputFeedback.tsx

```diff
@@ -1,23 +1,29 @@
-import { Icon, Info, Warning } from '@phosphor-icons/react';
+import { CheckCircle, Icon, Info, Warning } from '@phosphor-icons/react';
 import styled from 'styled-components';
 import Theme from 'theme';
 import { StyledP } from './common.styled';
 
-export type FeedbackVariant = 'info' | 'danger';
+export type FeedbackVariant = 'info' | 'danger' | 'plain' | 'plainIndented' | 'checkmark';
 
 const colors: Record<FeedbackVariant, string> = {
   info: Theme.colors.white_200,
   danger: Theme.colors.danger_light,
+  plain: Theme.colors.white_200,
+  plainIndented: Theme.colors.white_200,
```

Comment:
maybe this should be a `indentedNoIcon?: boolean` prop? 
so that any of the variants can replace the icon with an indent

Reply:
or just use 

noIcon?: boolean

and restyle this use where you're using it for bns names, because we don't want to bloat the ui-library with random exceptions that may not be used much in other places